swift/Samples/TicTacToe/Sources/Authentication/AuthenticationWorkflow.swift

```diff
@@ -111,62 +111,7 @@ extension AuthenticationWorkflow {
 
 // MARK: Workers
 
-extension AuthenticationWorkflow {
-
-    struct AuthorizingEmailPasswordWorker: Worker {
-
-        typealias Output = Action
-
-        var authenticationService: AuthenticationService
-        var email: String
-        var password: String
-
-        func run() -> SignalProducer<Output, Never> {
-            return authenticationService
-                .login(email: email, password: password)
-                .map({ response -> Action in
-                    return .authenticationSucceeded(response: response)
-                })
-                .flatMapError {
-                    SignalProducer(value: .authenticationError($0))
-                }
-        }
-
-        func isEquivalent(to otherWorker: AuthorizingEmailPasswordWorker) -> Bool {
-            return email == otherWorker.email
-            && password == otherWorker.password
-        }
-
-    }
-
-    struct AuthorizingTwoFactorWorker: Worker {
-
-        typealias Output = Action
-
-        var authenticationService: AuthenticationService
-        var intermediateToken: String
-        var twoFactorCode: String
-
-        func run() -> SignalProducer<Output, Never> {
-            return authenticationService
-                .secondFactor(
-                    token: intermediateToken,
-                    secondFactor: twoFactorCode)
-                .map {
-                    .authenticationSucceeded(response: $0)
-                }
-                .flatMapError {
-                    SignalProducer(value: .authenticationError($0))
-            }
-        }
-
-        func isEquivalent(to otherWorker: AuthenticationWorkflow.AuthorizingTwoFactorWorker) -> Bool {
```

Comment:
Need to make sure that we cover this either via a `key` argument to `asWorker` or the `key` argument to awaitResult(for:outputMap)`.